cli/js/web/url.ts

```diff
@@ -408,7 +409,7 @@ export class URLImpl implements URL {
 
   constructor(url: string | URL, base?: string | URL) {
     let baseParts: URLParts | undefined;
-    if (base) {
+    if (base != null) {
```

Comment:
What's the reason behind these changes? I for one prefer the current way, the intent is clear, you expect a _truthy_ value, on the other hand when using `!= null` it can either be a bug (=== should be used) or you expect a truthy value. 

Is there a style guide?